extensions/functions_comparison.yaml

```diff
@@ -214,3 +214,30 @@ scalar_functions:
         variadic:
           min: 2
         return: any1
+  -
+    name: "least"
+    description: >-
+      Returns the smallest value. Only return null if 'all' arguments evaluate to null.
+
+      String comparison is done in lexicographical ordering, one character at a time, from left to right.
```

Comment:
By lexicographic is the assumption the "C" locale?  The ordering could be different if an alternative locale is chosen.